components/LocationSelector.jsx

```diff
@@ -0,0 +1,62 @@
+var React = require('react');
+var Select = require('react-select');
+var Map = require('../components/map.jsx');
+
+var LocationSelector = React.createClass({
+  getInitialState: function() {
+    return {
+      location: null
+    };
+  },
+  render: function() {
+    return (
+      <Select
+       disabled={this.props.disabled}
+       placeholder={this.props.placeholder || "Type in a city or a country"}
```

Comment:
Ah. I see the attributes below do the same thing. No need to change now, just curious.